lntest/harness.go

```diff
@@ -573,6 +576,42 @@ func (n *NetworkHarness) StopNode(node *HarnessNode) error {
 	return node.stop()
 }
 
+//SaveProfilesPages hits profiles pages of all active nodes and writes it to disk
+//using a similar naming scheme as to the regular set of logs.
+func (n *NetworkHarness) SaveProfilesPages() {
+	for _, node := range n.activeNodes {
+		url := "http://localhost:%d/debug/pprof/goroutine?debug=1"
+
+		port := 9000 + node.NodeID
+
+		resp, err := http.Get(fmt.Sprintf(url, port))
+
+		if err != nil {
+			fmt.Println(err)
```

Comment:
In case of printing the errors to the screen, suggest you also print the node ID  and Name so it will be clear for which node there was a failure.

Reply:
Also, in such cases, suggest to use "continue"  as there is no much value to process current node after an error.

Reply:
Done.